docs/content/manual/pre-release/ha/ha-volume-migration.md

```diff
@@ -0,0 +1,31 @@
+---
+title: HA Volume Migration
+---
+
+1. Create a migratable volume:
+    1. Deploy a migratable StorageClass. e.g., https://github.com/longhorn/longhorn/blob/master/examples/rwx/storageclass-migratable.yaml
+    2. Create a PVC with access mode `ReadWriteMany` via this StorageClass.
+2. Attach a volume to a node and wait for volume running. Then write some data into the volume. Here I would recommend directly restoring a volume (set `fromBackup` in the StorageClass) and attach it instead.
+3. Start the migration by request attaching to another node for the volume.
+4. Trigger the following scenarios then confirm or rollback the migration:
+
+   --------------------------
+   | **#** | **Scenario**          | **Migration Confirmation** | **Migration Rollback** |
+   | ----- | --------------------- | -------------------------- | ---------------------- |
+   | 1     | New engine crash      | If the request is successfully sent before the migration engine & replicas being removed, the volume should be auto reattached after migration. <br> Otherwise, the request will be rejected until the volume is reattached and restarts the migration components | Should succeed |
+   | 2     | All new replica crash | If the request is successfully sent before the migration engine & replicas being removed, the volume should keep running (then may start rebuilding) after migration. <br> Otherwise, the request will be rejected until the volume is reattached and restarts the migration components | Should succeed |
+   | 3     | Old engine crash      | If the request is successfully sent before the volume becoming Faulted, the volume should keep running after migration. <br> Otherwise, the request will be rejected until the volume is reattached and restarts the migration components | Should succeed |
+   | 4     | All old replica crash | If the request is successfully sent before the volume becoming Faulted, the volume should keep running (then may start rebuilding) after migration. <br> Otherwise, the request will be rejected until the volume is reattached and restarts the migration components | Should succeed |
```

Comment:
If Auto-salvage is disabled, the volume will remain in faulted, right?

Reply:
Yes. But I didn't test this case, can you help check it?

Reply:
Yes, Checked it. If auto-salvage is disabled, the volume will remain in faulted state.